src/libServer/LookupServer.h

```diff
@@ -391,11 +391,9 @@ class LookupServer : public Server,
     auto fields = parseRawTxFields(rawTx);
 
     auto shards = m_mediator.m_lookup->GetShardPeers().size();
-    auto currentGasPrice =
-        m_mediator.m_dsBlockChain.GetLastBlock().GetHeader().GetGasPrice();
 
-    auto resp = CreateTransactionEth(fields, pubKey, shards, currentGasPrice,
-                                     m_createTransactionTarget);
+    auto resp =
+        CreateTransactionEth(fields, pubKey, shards, m_createTransactionTarget);
```

Comment:
Are you sure it is correct to remove the current Gas price and only check against the minimum?

Btw, I'm happy with the change as it is, and we might actually do it like that until we figure out our gas approach.  But please leave a TODO comment about that.

Reply:
I think you're right, I misunderstood that with minimum price for legacy transactions for some reason. Probably we need to check the price against `max(currentGasPrice * EVM_ZIL_SCALING_FACTOR, MIN_ETH_GAS_PRICE_IN_WEI)`. What do you think?

Reply:
Theoretically, yes.  But we also simultaneously need a separate gas price for EVM. (See https://zilliqa-jira.atlassian.net/browse/ZIL-4718).